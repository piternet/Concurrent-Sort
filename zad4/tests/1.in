5
95146
28389
10660
65977
28147
7872
88902
84454
16277
99229
