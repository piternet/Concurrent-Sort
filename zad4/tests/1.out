7872
10660
16277
28147
28389
65977
84454
88902
95146
99229
